.github/CODEOWNERS

```diff
@@ -1,17 +1,43 @@
 * @DataDog/datadog-ci
 
-src/commands
-src/commands/synthetics @DataDog/synthetics-worker
-src/commands/lambda @DataDog/serverless
-src/commands/sourcemaps @DataDog/source-code-integration
-src/commands/git-metadata @DataDog/source-code-integration
-*.md @DataDog/documentation
-src/commands/junit @DataDog/ci-app-tracers
-src/commands/dsyms @DataDog/rum-mobile
-src/helpers/ci.ts @DataDog/ci-app-tracers
-src/helpers/git.ts @DataDog/ci-app-tracers
-src/helpers/user-provided-git.ts @DataDog/ci-app-tracers
-src/helpers/__tests__/ci-env @DataDog/ci-app-tracers
-src/helpers/__tests__/ci.test.ts @DataDog/ci-app-tracers
-src/helpers/__tests__/git.test.ts @DataDog/ci-app-tracers
-src/helpers/__tests__/user-provided-git.test.ts @DataDog/ci-app-tracers
+
+# Commands
+
+## CI Visibility
+src/commands/junit   @DataDog/ci-app-libraries
+src/commands/metric  @DataDog/ci-app-libraries
+src/commands/tag     @DataDog/ci-app-libraries
+src/commands/trace   @DataDog/ci-app-libraries
+
+## RUM
+src/commands/dsyms         @DataDog/rum-mobile
+src/commands/react-native  @DataDog/rum-mobile
+
+## Serverless
+src/commands/lambda  @DataDog/serverless
+
+## Source Code Integration
+src/commands/git-metadata  @DataDog/source-code-integration
+src/commands/sourcemaps    @DataDog/source-code-integration
+
+## Synthetics
+src/commands/synthetics  @DataDog/synthetics-worker
+
+
+# Shared utilities
+
+## Git
+src/helpers/git  @DataDog/ci-app-libraries @DataDog/source-code-integration
```

Comment:
should this be 
```suggestion
src/helpers/git.ts  @DataDog/ci-app-libraries @DataDog/source-code-integration
```

Reply:
`src/helpers/git.ts` was split in multiple files in `src/helpers/git/` as part of https://github.com/DataDog/datadog-ci/pull/547